osx/ka-lite-remover.sh

```diff
@@ -166,7 +168,16 @@ if [ $IS_PREINSTALL == false ]; then
         echo "Answer no if you want to keep your KA-Lite data files."
         echo
         echo -n "Do you want the $KALITE_HOME directory to be deleted? (Yes/No) "
-        read remove_kalite
+        # Check if the second input argument in the script was passed.
+        if [ "$2" == "yes" ]; then
+            remove_kalite="yes"
+            syslog -s -l error "Auto confirm since $2 has value."
```

Comment:
Please rephrase this @arceduardvincent - what do you mean by "since $2 has value"?